tests/schema_registry/conftest.py

```diff
@@ -0,0 +1,350 @@
+    +--------+----------------------------------+-------+------------------------------+
+    | POST   | /subjects/conflict/versions      | 409*  | Incompatible Schema          |
+    +--------+----------------------------------+-------+------------------------------+
+    | POST   | /subjects/invalid/versions       | 42201 | Invalid Schema               |
+    +--------+----------------------------------+-------+------------------------------+
+    | POST   | /subjects/notfound               | 40401 | Subject not found            |
+    +--------+----------------------------------+-------+------------------------------+
+    | POST   | /subjects/schemanotfound         | 40403 | Schema not found             |
+    +--------+----------------------------------+-------+------------------------------+
+    | DELETE | /subjects/notfound               | 40401 | Subject not found            |
+    +--------+----------------------------------+-------+------------------------------+
+    | DELETE | /subjects/notfound/versions/[0-9]| 40401 | Subject not found            |
+    +--------+----------------------------------+-------+------------------------------+
+    | DELETE | /subjects/notfound/versions/404  | 40402 | Version not found            |
+    +--------+----------------------------------+-------+------------------------------+
+    | DELETE | /subjects/notfound/versions/422  | 42202 | Invalid version              |
+    +--------+----------------------------------+-------+------------------------------+
+    | GET    | /config/notconfig                | 40401 | Subject not found            |
+    +--------+----------------------------------+-------+------------------------------+
+    | PUT    | /config**                        | 42203 | Invalid compatibility level  |
+    +--------+----------------------------------+-------+------------------------------+
+    * POST /subjects/{}/versions does not follow the documented API error.
+    ** PUT /config reacts to a trigger in the body: - {"compatibility": "FULL"}
+
+    When evaluating special paths with overlapping trigger words the right most
+    keyword will take precedence.
+
+    i.e. Version not found will be returned for the following path.
+        /subjects/notfound/versions/404
+
+    The config endpoint has a special compatibility level "INVALID". This should
+    be used to verify the handling of in valid compatibility settings.
+
+    """
+    # request paths
+    schemas = re.compile("/schemas/ids/([0-9]*)")
+    subjects = re.compile("/subjects/?(.*)")
+    subject_versions = re.compile("/subjects/(.*)/versions/?(.*)")
+    compatibility = re.compile("/config/?(.*)")
+
+    # constants
+    SCHEMA_ID = 47
+    VERSION = 3
+    VERSIONS = [1, 2, 3, 4]
+    SCHEMA = 'basic_schema.avsc'
+    SUBJECTS = ['subject1', 'subject2']
+
+    # Counts requests handled per path by HTTP method
+    # {HTTP method: { path : count}
+    COUNTER = {'DELETE': {},
```

Comment:
nit: upper-case symbols are typically used for constants, while this here is not.